libglusterfs/src/iobuf.c

```diff
@@ -549,8 +595,8 @@ iobuf_get2(struct iobuf_pool *iobuf_pool, size_t page_size)
                     NULL);
             goto post_unlock;
         }
-
         iobuf_ref(iobuf);
+        iobuf->page_size = page_size;
```

Comment:
I think it's better to do this inside `__iobuf_get()`

Reply:
done